tests/pgautofailover_utils.py

```diff
@@ -473,7 +482,8 @@ def set_replication_quorum(self, replicationQuorum):
             Sets replication quorum via pg_autoctl
         """
 
-        set_command = [shutil.which('pg_autoctl'), 'set', 'node', '--pgdata', self.datadir,
+        set_command = [shutil.which('pg_autoctl'),
```

Comment:
can we hava PGAutoCtl class handle this too ?

Reply:
Good point, I did that in the follow-up commit, now pushed.